app/components/Notifier/Notifier.jsx

```diff
@@ -6,6 +6,7 @@ import PropTypes from 'prop-types';
 import styles from './notifier.css';
 import logger from 'logger';
 import { CLASSES } from 'appConstants';
+import AuthRequest from '../AuthRequest';
```

Comment:
We shouldn't be importing something so specifically SAFE in the browser domain. This should be abstracted/enabled via extensions if we need this.